src/views/registration-js/attendees/fields/datetime.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/datetime.php
+ *
+ * @since  TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Datetime
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 10dc47a647fab4b55a9c1ea98e173a62</sub>